docker/jenkins/Dockerfile.windows

```diff
@@ -22,7 +22,8 @@ RUN choco install -y cmake --installargs 'ADD_CMAKE_TO_PATH=""System""' --fail-o
   choco install -y -i ant; `
   choco install -y windows-sdk-10.1 --version 10.1.17134.12; `
   choco install -y 7zip; `
-  choco install -y ninja --version "1.10.0"
+  choco install -y ninja --version "1.10.0"; `
+  choco install -y python
```

Comment:
Only concern is that this will trigger this path in the container, which previously was only being used on dev machines (by design). It might be fine, but we might also want to make this conditional on whether we're running via jenkins like we do in some other places (though not for Windows yet AFAIK).

https://github.com/rstudio/rstudio/blob/076783392d7a27ca11d8af631b888311346573eb/dependencies/windows/install-dependencies.cmd#L208-L213

Reply:
That came from this PR of mine: https://github.com/rstudio/rstudio/pull/10080